ui/src/app/pages/signed-in/signed-in.tsx

```diff
@@ -159,22 +156,19 @@ export const SignedInImpl = (spinnerProps: WithSpinnerOverlayProps) => {
                   : styles.appContainer
               }
             >
-              {enableDemographicSurveyV2Redirect &&
-              pastSurveyDueDate &&
+              {pastSurveyDueDate &&
               !profileState.profile.demographicSurveyV2 &&
               !hasDismissedDemographicSurvey ? (
                 <DemographicSurveyPage
-                  routeData={{ title: 'Demographic Page' }}
+                  routeData={{ title: 'Demographic Survey' }}
                 />
               ) : (
                 <SignedInRoutes />
               )}
             </div>
           )}
       </FlexRow>
-      {!hideFooter && environment.enableFooter && (
```

Comment:
`enableFooter` has been true in all environments for a long time, so it has been removed.